docs/src/foundations/bindings_scopes.md

```diff
@@ -0,0 +1,118 @@
+# Bindings and Scopes
+
+How do we define symbols? And more importantly, how do we assign values to them? Let's address these
+questions now.
+
+```
+λ > a
+```
+
+Congratulations, we've defined our first symbol.
+
+```
+error[E02]: FreeIdentifier
+  ┌─ :1:1
+  │
+1 │ a
+  │ ^ a
+```
+
+Well, not so fast.
+
+As mentioned earlier in the book, the interpreter evaluates S-expressions that it encounters, and
+the symbol `a` is no exception. Since symbol evaluation involves substituting it with a bound value,
+and there is none bound to it, we get an error.
+
+Let's change that.
+
+### `define`
+
+This is how we bind values to symbols, using `(define symbol value)`:
+
+```scheme
+λ > (define a 1337)
+λ > a
+=> 1337
+```
+
+Here, we've created a _binding_ between the symbol `a` and a number literal. Consequently, further
+evaluation of `a` yields the elite integer. It's worth noting that regardless of whether `a` is
+bound and used across different expressions (in separate interpreter entries), it still retains its
+value. Why? Because the Steel session we're in introduces a _scope_, a context used in evaluation to
+resolve symbol bindings.
+
+Another example:
+
+```scheme
+λ > (define c (+ 1000 337))
+λ > c
+=> 1337
+```
+
+Although evaluation rules differ for `define` (symbol `c` is not evaluated, otherwise it would
+result in an error and make no sense), the _value_ part follows standard rules.
+
+### `let` and local scope `define`
+
+If `define` seems reminiscent of global variables to you, you're not alone. While it's suitable for
+defining function bindings (more on that later) or constants, there must be a better way for local
+bindings.
+
+Steel has two ways to do it: one is a Lisp classic, `let` bindings, and another one is
+Steel-specific local scope `define`.
```

Comment:
Not a deal breaker, but this isn't specific to steel, local defines are used in both Racket and Guile - as well as other schemes. I'm not 100% sure if it is part of the spec itself, I'd have to check.

Reply:
You're right, can see such examples in r6rs